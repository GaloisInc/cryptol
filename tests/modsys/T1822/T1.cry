module Test where

submodule S where
  submodule E where
    x = 0x02

submodule S where
