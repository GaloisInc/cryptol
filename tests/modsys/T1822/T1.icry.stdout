Loading module Cryptol
Loading module Cryptol
Loading module Test

[error] at :1:1--1:1
    Module not in scope: E
[error]
    Overlapping symbols defined:
    (at T1.cry:3:11--3:12, Test::S)
    (at T1.cry:7:11--7:12, Test::S)
